src/components/atoms/Typography/Typography.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactNode } from "react";
+import {
+  ChatTitleTypography,
+  ConversationPreviewAuthorTypography,
+  MessageContentTypography,
+  RegularTypography,
+  SectionTitleTypography,
+} from "./TypographyVariants";
+import { TypographyVariant } from ".";
```

Comment:
Dobrze rozumiem, że to importuje TypographyVariant z folderu, w którym jest ten plik (Typography.tsx)?

Reply:
Tak i nie. To importuje "TypographyVariant" z pliku "index", a w naszym przypadku "index.ts".
A że index.ts ma odniesienie do "TypographyVariant" to tak jak piszesz importuje do niego :)))